src/components/views/elements/ImageView.tsx

```diff
@@ -43,14 +43,14 @@ const ZOOM_COEFFICIENT = 0.0025;
 // If we have moved only this much we can zoom
 const ZOOM_DISTANCE = 10;
 
-interface IProps {
+export interface IProps {
```

Comment:
we shouldn't be exporting props interfaces, you can get at them using `ComponentProps<typeof ImageView>`